src/pykiso/tool/pykiso_to_pytest/cli.py

```diff
@@ -0,0 +1,168 @@
+
+    :param dic: dictionary to search in
+    :param keys: key path as list
+    :return: element at located position
+    """
+    for key in keys:
+        dic = dic[key]
+
+    if copy:
+        return dict(dic)
+
+    return dic
+
+
+def remove_key(dic: dict, key_path: Tuple[str, ...]) -> None:
+    """Remove dictionary element at given location.
+
+    :param dic: dictionary where the key shall be removed
+    :param key_path: key path as list
+    """
+    for key in key_path[:-1]:
+        dic = dic[key]
+    del dic[key_path[-1]]
+
+
+def get_imports(file: str) -> List[str]:
+    """Get all "from x import y" from a python script.
+
+    :param file: python file to read in
+    :return: all from imports
+    """
+
+    return [line for line in open(file, encoding="utf8") if re.match(r"from.*$", line)]
+
+
+@click.command()
+@click.argument("filename", type=click.Path(exists=True))
+@click.option(
+    "-d",
+    "--destination",
+    default="./conftest.py",
+    help="Output file path.",
+    show_default=True,
+)
+def main(filename: str, destination: str):
+    """Generate files from a pykiso config to use
+    pytest instead of pykiso tests.
+
+    :param filename: pykiso yaml file to convert
+    :param destination: destination file path
+    """
+    pykiso_config = config_parser.parse_config(filename)
+
+    template_loader = FileSystemLoader(searchpath=str(ROOT_PATH / "templates"))
+    template_env = Environment(loader=template_loader)
+    template_env.filters["format_value"] = format_value
+    template = template_env.get_template("conftest_template.jinja2")
+
+    # Grap import informations -> Channels and Auxiliaries
+    yaml_config_folder = pathlib.Path(filename).resolve().parent
+
+    import_info = set()
+
+    for _, value in pykiso_config["auxiliaries"].items():
+        import_info.add(value["type"])
+
+    for _, value in pykiso_config["connectors"].items():
+        import_info.add(value["type"])
+
+    module_imports = {}
+    for aux_con_type in import_info:
+        value, key = aux_con_type.split(":")[-2:]
+        if ".py" in value:
+            ext_file_imports = get_imports(yaml_config_folder / value)
+            for imp in ext_file_imports:
+                _, value, _, key = imp.strip().split(" ")
+
+        module_imports[key] = value
+
+    # Check if we have a proxy auxiliary
+    key_path = find_string_in(pykiso_config, "ProxyAuxiliary")
+    proxy_aux_config = None
+    if key_path:
+        proxy_aux_config = nested_get(pykiso_config, key_path[:-1], copy=True)
+        remove_key(pykiso_config, key_path[:-1])
+        proxy_aux_config["name"] = key_path[1]
+
+    # Use jinja to render conftest
+    conftest = template.render(
+        dict_item=pykiso_config,
+        proxy_aux_config=proxy_aux_config,
+        module_imports=module_imports,
+    )
+
+    # Sort imports with isort and use black to format the code
+    conftest = format_str(isort.code(conftest), mode=FileMode())
```

Comment:
Would it also make sense to run black?

Reply:
its already blacked (:
from black import FileMode, format_str

Reply:
The imports are blacked, but the generated code isn't ;) 
But in the end as the template is well made, you probably don't need to run black on it

Reply:
? why do you assume that only the imports are blacked ? 
Imports are isorted and the rest is blacked.. -> black.format_str()

Reply:
Ah my bad, way too used to see the entire black.format_str statement, I might need glasses :D